### ISSUE TITLE
Rename a context or project globally

### ISSUE THREAD

**Issue by mNantern** [QTodoTxt/QTodoTxt#51]
It may be useful to rename a project or a context in the left tree and change the value across all tasks.

**Comment by kmicc** [comment]
Yes, it would be nice to have such a feature. But as a workaround, you can simply open the todo-text-file in an arbitrary text editor and use the search&replace function.
